plume-models/src/schema.rs

```diff
@@ -183,6 +183,7 @@ table! {
         is_admin -> Bool,
         summary -> Text,
         email -> Nullable<Text>,
+        custom_domain -> Nullable<Text>,
```

Comment:
Why is custom_domain `NOT NULL` in migrations but `Nullable` here?

Reply:
does this mean we can resolve this?

Reply:
You can either leave this open or resolve and mark [this](https://github.com/Plume-org/Plume/pull/340/files#diff-5b7775a42e5a98eee192a505e07337baR2) and [that](https://github.com/Plume-org/Plume/pull/340/files#diff-03a9c6038bd075d7cb69055575b8f298R2) instead